index.js

```diff
@@ -908,6 +905,7 @@ module.exports = class Autobase extends ReadyResource {
     this.fastForwardTo = null
     this._queueFastForward()
 
+    this.wasActiveIndexer = this._isActiveIndexer()
```

Comment:
need to init this to false in constructor. I think we should make it an _ prop also

Reply:
ahh my bad, it shouldn't be here